src/javascript/ast-value.ts

```diff
@@ -30,12 +30,27 @@ function literalToValue(literal: estree.Literal): LiteralValue {
 /**
  * converts unary to string
  */
-function unaryToValue(unary: estree.UnaryExpression): string {
-  const argValue = expressionToValue(unary.argument);
-  if (argValue === undefined) {
-    return;
+function unaryToValue(unary: estree.UnaryExpression): LiteralValue {
```

Comment:
who's calling this function? Is the call site going to be ok with return values changing from `"!0"` to `true`? It doesn't look like the intent of the current version is to actually evaluate the operator.